state-chain/pallets/cf-ingress-egress/src/tests.rs

```diff
@@ -2120,7 +2120,8 @@ fn tainted_transactions_expire_if_not_witnessed() {
 		let tx_id = DepositDetails::default();
 		let expiry_at = System::block_number() + TAINTED_TX_EXPIRATION_BLOCKS as u64;
 
-		TaintedTransactions::<Test>::insert(BROKER, tx_id.clone(), expiry_at);
+		TaintedTransactions::<Test>::insert(BROKER, tx_id.clone(), ());
+		TxExpiresAt::<Test>::insert(expiry_at, vec![(BROKER, tx_id.clone())]);
```

Comment:
We should test using the real public methods, or at least with the `_inner` method.